code/headers/hardware_usart.hpp

```diff
@@ -29,6 +29,30 @@ namespace r2d2 {
     public:
         hardware_usart_c(unsigned int baudrate, uart_ports_c &usart_port);
 
+        /// console character output function
```

Comment:
all below comment are not valid with the style guide use /* etc see styleguide

Reply:
This is actualy not specified in the style guide, it is never named whether to use /// or /*

Reply:
https://github.com/R2D2-2019/R2D2-2019/wiki/Style-Guide#doxygen-and-comments